Upstream-Status: Pending

The prototype of function internall_error has changed. And one place in the code was not synced up with the new prototype, causing build error. This patche fixes that.

Nitin A Kamble <nitin.a.kamble@intel.com>
2010/11/12

Index: gdb-7.2/gdb/gdbserver/linux-i386-ipa.c
===================================================================
--- gdb-7.2.orig/gdb/gdbserver/linux-i386-ipa.c
+++ gdb-7.2/gdb/gdbserver/linux-i386-ipa.c
@@ -178,7 +178,7 @@ supply_static_tracepoint_registers (stru
 	    }
 	    break;
 	  default:
-	    internal_error ("unhandled register size: %d",
+	    internal_error (__FILE__, __LINE__, "unhandled register size: %d",
 			    i386_st_collect_regmap[i].size);
 	  }
       }
